CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.60.5](https://github.com/serverless/serverless/compare/v1.60.0...v1.60.5) (2020-01-03)
```

Comment:
The comparison range needs to be updated.

Reply:
argh.. again :) Well spotted.

btw. I have a bit better idea for handling release chains, which will make all releases linear (so no such issues will occur). I'll try to describe it in some gh issue shortly.

Fixed